spring-cloud-stream-binder-kafka/pom.xml

```diff
@@ -53,13 +48,24 @@
 		<dependency>
 			<groupId>org.springframework.kafka</groupId>
 			<artifactId>spring-kafka</artifactId>
-			<version>${spring-kafka.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework.integration</groupId>
-			<artifactId>spring-integration-kafka</artifactId>
-			<version>${spring-integration-kafka.version}</version>
 		</dependency>
+<!-- begin temporary -->
+		<!--<dependency>-->
+			<!--<groupId>org.springframework.integration</groupId>-->
+			<!--<artifactId>spring-integration-core</artifactId>-->
+			<!--<version>4.3.12.BUILD-SNAPSHOT</version>-->
+		<!--</dependency>-->
+		<!--<dependency>-->
+			<!--<groupId>org.springframework.integration</groupId>-->
+			<!--<artifactId>spring-integration-jmx</artifactId>-->
+			<!--<version>4.3.12.BUILD-SNAPSHOT</version>-->
+		<!--</dependency>-->
+		<!--<dependency>-->
+			<!--<groupId>org.springframework.integration</groupId>-->
+			<!--<artifactId>spring-integration-test</artifactId>-->
+			<!--<version>4.3.12.BUILD-SNAPSHOT</version>-->
+		<!--</dependency>-->
+<!-- end temporary -->
```

Comment:
We don't need this block any more I guess. 
We really now rely on the transitive dependency from the SCSt Core.